sdk/php/dev/src/PhpSdkDev.php

```diff
@@ -82,6 +91,12 @@ public function formatStdout(Directory $source): string
 
     private function base(Directory $source): Container
     {
+        if (!is_null($this->base_container)) {
+            return $this
+                ->base_container
+                ->withMountedDirectory(self::SDK_ROOT, $source);
+        }
+
         return dag()
```

Comment:
```suggestion
        return $this->baseContainer ?? dag()
```

Since this is how you're getting the base:
https://github.com/dagger/dagger/blob/fca16c064545c9e2f8c1d8c9e99bb1053af29bbc/.dagger/sdk_php.go#L75-L79

I don't think you'll need to mount the directory, as that function already does it:

https://github.com/dagger/dagger/blob/fca16c064545c9e2f8c1d8c9e99bb1053af29bbc/.dagger/sdk_php.go#L179-L193

The `$this->baseContainer ?? other thing` is shorthand for: "use `$this->baseContainer`, but if it's null, use `other thing`. If you want to understand it more [check here](https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op)